### PR TITLE
Implement py-cinema-visit

### PR DESCRIPTION
**Pull request by SevKrok**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list,
+            cleaning_staff: Cleaner
+    ) -> None:
```
Comment (Wonsky1) [inline app/cinema/hall.py:13]: Add correct type annotiation for "customers"

**Review by Wonsky1 — CHANGES_REQUESTED** [review]

**Review by Wonsky1 — APPROVED** [review]

File: app/cinema/hall.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list[Customer],
+            cleaning_staff: Cleaner
```
Comment (LiudmylaKulinchenko) [inline app/cinema/hall.py:13]: too many spaces for indentation

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list,
+        hall_number: int,
+        cleaner: str,
+        movie: str
```
Comment (LiudmylaKulinchenko) [inline app/main.py:11]: too many spaces for indentation

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+) -> None:
+    cinema_bar = CinemaBar()
```
Comment (LiudmylaKulinchenko) [inline app/main.py:13]: No need to create an instance to call static method

File: app/main.py [hunk 5]
```diff
@@ -1,6 +1,24 @@
...
+    hall_number = CinemaHall(hall_number)
+    customers = [
+        Customer(customer.get("name"), customer.get("food"))
+        for customer in customers
+    ]
+    cleaner = Cleaner(cleaner)
```
Comment (LiudmylaKulinchenko) [inline app/main.py:19]: Assign variable in the top of function, move it to hall_number assigning

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 6]
```diff
@@ -1,6 +1,21 @@
...
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list, hall_number: int,
+                 cleaner: str, movie: str
+                 ) -> None:
```
Comment (LiudmylaKulinchenko) [inline app/main.py:9]: Fix formatting: brackets must be located in one of two options: open and start with the text or have a line break between the text
Also one parameter in one line to increase readability

File: app/cinema/hall.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list,
```
Comment (LiudmylaKulinchenko) [inline app/cinema/hall.py:11]: customers: list[Customer] was fine

File: app/cinema/hall.py (hunk 1 above)
Comment (LiudmylaKulinchenko) [inline app/cinema/hall.py:13]:
```suggestion
    def movie_session(
        self,
        movie_name: str,
        customers: list,
        cleaning_staff: Cleaner
    ) -> None:
```

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 👏

**Pull request closed without merging**